tests/Unit/Client/Image/ImageUrlUploadTest.php

```diff
@@ -17,12 +17,25 @@ final class ImageUrlUploadTest extends TestCase
      */
     public function productImageUpload_withURL(): void
     {
-        $imageUrl = 'https://example.com/image.png';
+        $allowedImageType = [
+            'gif'  => 'GIF',
+            'jpeg' => 'JPEG',
+            'jpg'  => 'JPEG',
+            'png'  => 'PNG',
+            'bmp'  => 'BMP',
+        ];
+
+        $imageUrl       = 'https://example.com/image.png';
+        $imageFormat    = $allowedImageType[
```

Comment:
Please make sure the code style is inline with the other files:
(alignment equal signs)

run `make cs-fix` to fix automatically